tls/s2n_signature_algorithms.c

```diff
@@ -97,14 +95,12 @@ static int s2n_choose_sig_scheme(struct s2n_connection *conn, struct s2n_sig_sch
 int s2n_get_and_validate_negotiated_signature_scheme(struct s2n_connection *conn, struct s2n_stuffer *in,
                                              struct s2n_signature_scheme *chosen_sig_scheme)
 {
+    notnull_check(conn);
```

Comment:
Is this necessary?

Reply:
`conn` is de-referenced within the `s2n_connection_get_signature_preferences` call within this function. We also do a notnull_check within that function call for conn, so by adding this check we are catching the error earlier.